core/src/main/scala/pureconfig/error/ConfigReaderFailure.scala

```diff
@@ -18,8 +18,10 @@ trait ConfigReaderFailure {
 
   /**
    * A human-readable description of the failure.
+   *
+   * @param indentSize the number of characters to use for any indentation applied inside the description
    */
-  def description: String
+  def description(indentSize: Int): String
```

Comment:
I'm OK with always going with 2 spaces, but then I'd rather disallow configuring the indentation size in the `prettyPrint` method of `ConfigReaderFailures`, so that the output is consistent. Did that in 6f10a2f.